netlify.toml

```diff
@@ -0,0 +1,4 @@
+[[redirects]]
+  from = "/*"
+  to = "/index.html"
```

Comment:
Why `index.html` 🤔

Reply:
`index.html` is where your app runs and its router will be able to process all requested routes.